### ISSUE TITLE
FNX-4454 ⁃ [Bug] Nightly 200514 crashes when opening a website and trying to tap/scroll

### ISSUE THREAD

**Issue by ksy36** [mozilla-mobile/fenix#10659]
## Steps to reproduce
Open any site, for example https://webcompat.com/issues/new and try to scroll down or tap anywhere

### Expected behavior
scrolling is performed 

### Actual behavior
crash 

### Device information
SM-G900M 

* Android device: 6.0.1 MMB29M
* Fenix version: 200514

Logcat info: https://gist.github.com/ksy36/6868b03b6809838854dd8bf9dc2ad34d
in `about:crashes` the crash is displayed as `java.lang.RuntimeException: java.lang.reflect.InvocationTargetException`

This happens on any website I've tried to visit

**Comment by ksy36** [comment]
This might be a duplicate of https://github.com/mozilla-mobile/fenix/issues/10647 (the gif in https://github.com/mozilla-mobile/fenix/issues/10647#issuecomment-628603371 illustrates the same problem)

**Comment by softvision-oana-arbuzov** [comment]
I was also able to reproduce it on YouTube, even with a clean profile.
URL:  https://m.youtube.com/watch?v=nJ7zozOm8lo

![AppCrashScroll](https://user-images.githubusercontent.com/12184325/82026156-04a15280-969b-11ea-8d13-e7eecaecbc29.gif)

Note: 
1. Reproducible: 10/10
2. For me the crash occurs when touching the screen at any position on the page.
3. Attached crash log:
[Crashlog.txt](https://github.com/mozilla-mobile/fenix/files/4633054/Crashlog.txt)

**Tested with:**
Browser / Version: Firefox Preview Nightly 200514 (🦎 78.0a1-20200513094918)
Operating System:  Samsung Galaxy S6 Edge (Android 7.0) - 1440 x 2560 pixels (~577 ppi pixel density)

**Comment by mcarare** [comment]
From stack trace, this is a duplicate of https://github.com/mozilla-mobile/fenix/issues/10647. Also, a fix has already been submitted in AC: https://github.com/mozilla-mobile/android-components/pull/6979 and will be tested in https://github.com/mozilla-mobile/fenix/issues/10647 once the change lands in Fenix.
